### ISSUE TITLE
Add setting to hide "Mark all as read"

### ISSUE THREAD

**Issue by albin-engstrom** [inbasic/ignotifier#686]
I've used this great extension for many years and have twice, although not recently, accidentally clicked "Mark all as read" which is quite a hassle when one is dealing with many unread emails for one reason or another. Especially when spread out over time with many read ones in-between.

It's close to "Mark as read" which I click often, so just a slight slip of the hand when clicking can, and has, lead to undesirable results.

It would be lovely to simply have a setting to hide the "Mark all as read" button to eliminate the risk and worry, I personally never use it

**Comment by inbasic** [comment]
On the options page, use the custom CSS rule for top panel section:

```css
[name=read-all] {display: none}
```

**Comment by albin-engstrom** [comment]
Thank you, although I can't find any custom CSS options on the options page.
And searching for "CSS" on the page yields nothing, so I don't think I've missed it.

Only these sections are there; Timings, Gmail, Notifications, Tab Opening, Toolbar, Miscellaneous and Plug-ins.
And none of those contain anything related to CSS.

I am using V2 version 1.1.9 on Firefox on Linux.

**Comment by inbasic** [comment]
Right! I though this feature is already published. Will be available on `1.2.0`

**Comment by albin-engstrom** [comment]
Excellent!
